README.md

```diff
@@ -1,25 +1,57 @@
-# README
+# Blog App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is a A rails application that allows users to create, like and comment on posts. It includes user authentication and data is preserved using postgresql.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby on Rails
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+- Ruby runtime environment
+- Postgresql
+- Rails
 
-* How to run the test suite
+### Setup
 
-* Services (job queues, cache servers, search engines, etc.)
+- Clone the repository using `git clone git@github.com:amejid/blog-app.git`
```

Comment:
- [ ] [optional] Kindly check on the clone repository it's linked to the wrong repository.

Reply:
Thanks, I'll fix that